hudi-timeline-service/src/main/java/org/apache/hudi/timeline/service/RequestHandler.java

```diff
@@ -202,6 +204,9 @@ public void stop() {
     if (markerHandler != null) {
       markerHandler.stop();
     }
+    if (asyncResultService != null) {
+      asyncResultService.shutdown();
```

Comment:
`shutdown` or `shutdownNow`?

Reply:
Shutdown is enough, it has not submitted long-term jobs that need to be interrupted, we can just exit smoothly